### PR TITLE
chore(multi-region): add operational simplification from Zeebe 8.6.

### PR DESCRIPTION
**Pull request by Langleu**
## Description

related to https://github.com/camunda/team-infrastructure-experience/issues/325 and https://github.com/camunda/product-hub/issues/2173.

This ticket contains the simplification of the dual-region operational procedure based on the new 8.6 changes in Zeebe.

It reworks for next (8.6):
- the pictures of the operational procedure
- the steps that need to be taken
- the concept page including RPO / RTO
- some smaller fixes here and there

## When should this change go live?

- [ ] This is a bug fix, security concern, or something that needs **urgent release support**.
- [ ] This is already available but undocumented and should be released within a week.
- [ ] This on a **specific schedule** and the assignee will coordinate a release with the DevEx team. (apply `hold` label or convert to draft PR)
- [x] This is part of a scheduled alpha or minor. (apply alpha or minor label)
    - either alpha5 or latest stable would be nice.
- [ ] There is **no urgency** with this change and can be released at any time.

## PR Checklist

- [ ] My changes are for an **already released minor** and are in `/versioned_docs` directory.
- [x] My changes are for the **next minor** and are in `/docs` directory (aka `/next/`).

- [x] My changes require an [Engineering review](https://github.com/camunda/camunda-docs/blob/main/howtos/documentation-guidelines.md#review-process), and I've assigned the Engineering DRI or delegate.
- [x] My changes require a [technical writer review](https://github.com/camunda/camunda-docs/blob/main/howtos/documentation-guidelines.md#review-process), and I've assigned @camunda/tech-writers as a reviewer.

### CODE REVIEW

**Comment by Langleu** [comment]
The change is intentional since 8.5 still uses the old procedure while was reworked for next (8.6)

**Comment by Langleu** [comment]
Great to hear @conceptualshark 🙇 🚀!
Since it's a change of an existing document, I think it would be okay to have your review already!
The only thing I could imagine is that maybe a step has to be moved, but the content itself will remain.

File: docs/self-managed/operational-guides/multi-region/dual-region-ops.md [hunk 1]
```diff
@@ -605,7 +608,7 @@ kubectl --context $CLUSTER_SURVIVING get deployments $HELM_RELEASE_NAME-operate
 # camunda-tasklist   0/0     0            0           23m
 ```
 
-For the Zeebe Elasticsearch exporters, there's currently no API available to confirm this. Only the response code of `204` indicates a successful disabling.
+For the Zeebe Elasticsearch exporters, there's currently no API available to confirm this. Only the response code of `204` indicates a successful disabling. This is a synchronous operation.
```
Comment (deepthidevaki) [inline docs/self-managed/operational-guides/multi-region/dual-region-ops.md:611]: You can also link the api docs above http://localhost:3000/docs/next/self-managed/zeebe-deployment/operations/management-api/#exporting-api

File: docs/self-managed/operational-guides/multi-region/dual-region-ops.md [hunk 2]
```diff
@@ -1000,38 +917,22 @@ desired={<Thirteen viewBox="140 40 680 500" />}
...
-  --namespace $CAMUNDA_NAMESPACE_RECREATED \
-  -f camunda-values.yml \
-  -f $REGION_RECREATED/camunda-values.yml \
-  --set global.multiregion.installationType=failBack
+kubectl --context $CLUSTER_SURVIVING port-forward services/$HELM_RELEASE_NAME-zeebe-gateway 9600:9600 -n $CAMUNDA_NAMESPACE_SURVIVING
+curl -XPOST 'http://localhost:9600/actuator/exporters/elasticsearchregion1/enable' -H 'Content-Type: application/json' -d '{"initializeFrom" : "elasticsearchregion0"}'
 ```
 
-3. Reactivate the exporters by sending the API activation request via the Zeebe Gateway:
+2. Reactivate the exporters by sending the API activation request via the Zeebe Gateway:
```
Comment (deepthidevaki) [inline docs/self-managed/operational-guides/multi-region/dual-region-ops.md:935]: It would be better to split this into a new step. We should resume only after the exporter has enabled. Enabling exporter usually finishes faster, but it is not a synchronous operation. So the process should be as follows.

1. enable exporter
2. verify it is enabled
3. Resume exporting

File: docs/self-managed/operational-guides/multi-region/dual-region-ops.md [hunk 3]
```diff
@@ -1042,121 +943,179 @@ curl -i localhost:9600/actuator/exporting/resume -XPOST
...
 
-Port-forwarding the Zeebe Gateway via `kubectl` and printing the topology should reveal that the failover brokers are missing:
+  </summary>
+</details>
 
-```bash
-kubectl --context $CLUSTER_SURVIVING port-forward services/$HELM_RELEASE_NAME-zeebe-gateway 26500:26500 -n $CAMUNDA_NAMESPACE_SURVIVING
-zbctl status --insecure --address localhost:26500
-```
+For the Zeebe Elasticsearch exporters, there's currently no API available to confirm this. Only the response code of `204` indicates a successful resumption. This is a synchronous operation.
```
Comment (deepthidevaki) [inline docs/self-managed/operational-guides/multi-region/dual-region-ops.md:986]:
```suggestion
For the reactivating the exporters, there's currently no API available to confirm this. Only the response code of `204` indicates a successful resumption. This is a synchronous operation.
```

File: docs/self-managed/concepts/multi-region/dual-region.md [hunk 4]
```diff
@@ -196,12 +190,12 @@ The **Recovery Time Objective (RTO)** is the time to restore services to a funct
 
 For Zeebe the **RPO** is **0**.
 
-For Operate and Tasklist the **RPO** is close to **0** for critical data due to the previously mentioned small chance of data loss in Elasticsearch during the failback procedure.
+For Operate and Tasklist the **RPO** is **0**.
```
Comment (deepthidevaki) [inline docs/self-managed/concepts/multi-region/dual-region.md:193]: Can be combined with the previous sentence.

File: docs/self-managed/concepts/multi-region/dual-region.md [hunk 5]
```diff
@@ -196,12 +190,12 @@ The **Recovery Time Objective (RTO)** is the time to restore services to a funct
...
 
 The **RTO** can be considered for the **failover** and **failback** procedures, both resulting in a functional state.
 
-- **failover** has an **RTO** of **15-20** minutes to restore a functional state, excluding DNS considerations.
-- **failback** has an **RTO** of **25-30 + X** minutes to restore a functional state. Where X is the time it takes to back up and restore Elasticsearch, which is highly dependent on the setup and chosen [Elasticsearch backup type](https://www.elastic.co/guide/en/elasticsearch/reference/current/snapshots-register-repository.html#ess-repo-types).
+- **failover** has an **RTO** of **< 1** minute to restore a functional state, excluding DNS considerations.
+- **failback** has an **RTO** of **5 + X** minutes to restore a functional state. Where X is the time it takes to back up and restore Elasticsearch, which is highly dependent on the setup and chosen [Elasticsearch backup type](https://www.elastic.co/guide/en/elasticsearch/reference/current/snapshots-register-repository.html#ess-repo-types).
```
Comment (deepthidevaki) [inline docs/self-managed/concepts/multi-region/dual-region.md:198]: :question: What is the reason for 5m?

**Review by deepthidevaki — DISMISSED** [review]
:rocket:  

One minor thing to change:

In Failover Step 1, it is not necessary to ensure network isolation between two regions anymore. Previously, we had to do it because we were recreating brokers with the same id on the other region which could create conflicts if the network was not isolated and both the old and new brokers exists at the same time.

File: docs/self-managed/concepts/multi-region/dual-region.md [hunk 6]
```diff
@@ -129,11 +127,7 @@ In the event of a total active region loss, the following data will be lost:
   - Role Based Access Control (RBAC) does not work.
 - Optimize is not supported.
   - This is due to Optimize depending on Identity to work.
-- Connectors are not supported.
-  - This is due to Connectors depending on Operate to work for inbound Connectors and potentially resulting in race condition.
-- During the failback procedure, there’s a small chance that some data will be lost in Elasticsearch affecting Operate and Tasklist.
-  - This **does not** affect the processing of process instances in any way. The impact is that some information about the affected instances might not be visible in Operate and Tasklist.
-  - This is further explained in the [operational procedure](./../../operational-guides/multi-region/dual-region-ops.md?failback=step2#failback) during the relevant step.
+- Connectors can be deployed alongside but ensure to understand idempotency based on [the described documentation](../../../components/connectors/use-connectors/inbound.md#creating-the-connector-event).
```
Comment (conceptualshark) [inline docs/self-managed/concepts/multi-region/dual-region.md:130]: Could this be more declarative regarding support?

```suggestion
- Connectors are supported, but may require special considerations regarding [idempotency](../../../components/connectors/use-connectors/inbound.md#creating-the-connector-event).
```
It's not clear to me from the linked documentation why idempotency would be more of a concern in this setup, but that may be my lack of familiarity. Is it _always_ a concern in this setup? Should they always use the message start event element? Some clarity here would be great.

File: docs/self-managed/concepts/multi-region/dual-region.md (hunk 5 above)
Comment (conceptualshark) [inline docs/self-managed/concepts/multi-region/dual-region.md:198]:
```suggestion
- **failback** has an **RTO** of **5 + X** minutes to restore a functional state, where X is the time it takes to back up and restore Elasticsearch. This timing is highly dependent on the setup and chosen [Elasticsearch backup type](https://www.elastic.co/guide/en/elasticsearch/reference/current/snapshots-register-repository.html#ess-repo-types).
```

File: docs/self-managed/operational-guides/multi-region/dual-region-ops.md [hunk 7]
```diff
@@ -44,12 +44,20 @@ Running dual-region setups requires the users to be able to detect any regional
...
 :::
 
+:::info
+
+The operational procedure was reworked in Camunda 8.6 to be more straightforward and reliable.
+
+The old procedure of 8.5 [as outlined](../../../../versioned_docs/version-8.5/self-managed/operational-guides/multi-region/dual-region-ops.md) is still compatible with Camunda 8.6 but compatibility will be removed in 8.7.
+
+:::
+
```
Comment (conceptualshark) [inline docs/self-managed/operational-guides/multi-region/dual-region-ops.md:54]:
```suggestion
:::info
This procedure has been updated in the Camunda 8.6 release. The procedure used in Camunda 8.5 has been deprecated, and compatibility will be removed in the 8.7 release.
:::
```
I'm so-so on if you'd like to include the link to the old version. If we're deprecating that method, I don't necessarily see the value in it, unless there are circumstances where the older procedure would be required. 

Can we move this up to the top of the page as well, for visibility?

File: docs/self-managed/operational-guides/multi-region/dual-region-ops.md [hunk 8]
```diff
@@ -177,82 +183,82 @@ Due to the Zeebe data replication, no data has been lost.
...
+You will port-forward the `Zeebe Gateway` in the surviving region to the local host to interact with the Gateway.
 
-In the case when your **Region 0** was lost, instead go to the folder [aws/dual-region/kubernetes/region1](https://github.com/camunda/c8-multi-region/blob/main/aws/dual-region/kubernetes/region1/) for the `camunda-values-failover.yml` file.
+The following alternatives to port-forwarding are possible:
 
-The chosen `camunda-values-failover.yml` requires adjustments before installing the Helm chart and the same has to be done for the base `camunda-values.yml` in `aws/dual-region/kubernetes`.
+- if it's exposed to the outside, one can skip port-forwarding and use the URL directly
+- one can use, for example, the Elasticsearch pod to exec into it and curl from there
```
Comment (conceptualshark) [inline docs/self-managed/operational-guides/multi-region/dual-region-ops.md:195]:
```suggestion
- one can `exec` into an existing pod (such as Elasticsearch), and `curl` from there
```

^ Is this a correct rephrasing? Some clarity + command backtics.

File: docs/self-managed/operational-guides/multi-region/dual-region-ops.md [hunk 9]
```diff
@@ -177,82 +183,82 @@ Due to the Zeebe data replication, no data has been lost.
...
-  -f $REGION_SURVIVING/camunda-values-failover.yml
+kubectl --context $CLUSTER_SURVIVING port-forward services/$HELM_RELEASE_NAME-zeebe-gateway 9600:9600 -n $CAMUNDA_NAMESPACE_SURVIVING
 ```
 
-#### Verification
-
-The following command will show the deployed pods of the failover namespace.
-
-Only the minimal amount of brokers required to restore the quorum will be deployed in the failover installation. For example, if `clusterSize` is eight, two Zeebe brokers will be deployed in the failover installation instead of the normal four. This is expected.
+3. Based on the [Cluster Scaling APIs](../../zeebe-deployment/operations/cluster-scaling.md) you will send a request to the Zeebe Gateway to redistribute the load to the remaining brokers, thereby removing the lost brokers.
```
Comment (conceptualshark) [inline docs/self-managed/operational-guides/multi-region/dual-region-ops.md:252]:
```suggestion
3. Based on the [Cluster Scaling APIs](../../zeebe-deployment/operations/cluster-scaling.md), send a request to the Zeebe Gateway to redistribute the load to the remaining brokers, thereby removing the lost brokers.
```

**Review by conceptualshark — CHANGES_REQUESTED** [review]
A few questions/suggestions, but otherwise this upgrade looks great!

File: docs/self-managed/concepts/multi-region/dual-region.md (hunk 5 above)
Comment (Langleu) [reply docs/self-managed/concepts/multi-region/dual-region.md:198]: I took the estimate from our automated tests for the procedure and comes mostly from installing C8.
First install of C8 and waiting for Elastic to be ready is roughly 2-3 minutes.
Later installing Operate / Tasklist is another minute.
In-between are the steps that need to be taken when automated of roughly a minute. (It's automated, so a manual execution will even take longer).

Comment (Langleu) [reply docs/self-managed/concepts/multi-region/dual-region.md:198]: Since RTO contains the time to recover as well, it's roughly 5 minutes from my side.
I'm open to other suggestions if you think it should be more or less.

File: docs/self-managed/operational-guides/multi-region/dual-region-ops.md (hunk 1 above)
Comment (Langleu) [reply docs/self-managed/operational-guides/multi-region/dual-region-ops.md:611]: linked it where it's executed.

File: docs/self-managed/operational-guides/multi-region/dual-region-ops.md [hunk 10]
```diff
@@ -328,127 +345,83 @@ desired={<Seven viewBox="140 40 680 500" />}
...
+2. List all exporters to find the corresponding ID. Alternatively, you can check your `camunda-values.yml` file, which lists the exporters as those had to be configured explicitly.
 
 ```bash
-helm upgrade $HELM_RELEASE_NAME camunda/camunda-platform \
-  --version $HELM_CHART_VERSION \
-  --kube-context $CLUSTER_SURVIVING \
-  --namespace $CAMUNDA_NAMESPACE_SURVIVING \
-  -f camunda-values.yml \
-  -f $REGION_SURVIVING/camunda-values.yml
+curl -XGET 'http://localhost:9600/actuator/exporters'
```
[the remark this answers is not in the document]
Comment (Langleu) [reply docs/self-managed/operational-guides/multi-region/dual-region-ops.md:367]: great! I left a todo in the text where we should probably link it and only have a wrong placeholder atm.

File: docs/self-managed/concepts/multi-region/dual-region.md (hunk 6 above)
Comment (Langleu) [reply docs/self-managed/concepts/multi-region/dual-region.md:130]: I don't want to use the word "supported" atm since we have an extra ticket to actually create tests / manifest adjustments etc. to officially support it.
It's just that it isn't a limitation anymore but can be done under premise to understand message idempotency.

I've added a subsentence on why it's important.
TL;DR - multiple connectors instances + message idempotency = no duplicated events.
Without it = may end up triggering things multiple times.
Same applies to a single region setup, just that in multi-region by default we have two connector instances.

File: docs/self-managed/operational-guides/multi-region/dual-region-ops.md (hunk 7 above)
Comment (Langleu) [reply docs/self-managed/operational-guides/multi-region/dual-region-ops.md:54]: I'm fine not including the link, a person can just use the navbar on top to quickly switch the version.
It was just for people to have a grace period to readjust their own documentation.

Sure moved it to the top.

File: docs/self-managed/operational-guides/multi-region/dual-region-ops.md (hunk 8 above)
Comment (Langleu) [reply docs/self-managed/operational-guides/multi-region/dual-region-ops.md:195]: sure, I also linked to the official k8s exec docs on the command.

**Comment by Langleu** [comment]
Thanks @deepthidevaki and @conceptualshark for the review 🙇.

@deepthidevaki:
- Good note on the isolation, I've added that information to the 8.5 procedure and removed it from 8.6

Besides that addressed all points except for the two that are left open, which we may have to realign on.

File: docs/self-managed/concepts/multi-region/dual-region.md (hunk 5 above)
Comment (deepthidevaki) [reply docs/self-managed/concepts/multi-region/dual-region.md]: Thanks. I'm wondering if we have to specify the 5 minutes. Can we say it in general that it depends on the ES backup and restore + some additional time that needs to install and start C8 components in the new region? 5 minutes seems arbitrary and can also depends on whether it is manual, automated etc.

File: docs/self-managed/concepts/multi-region/dual-region.md (hunk 6 above)
Comment (conceptualshark) [reply docs/self-managed/concepts/multi-region/dual-region.md:130]: This works for me then. 👍 The additional messaging helps!

**Review by conceptualshark — DISMISSED** [review]

File: docs/self-managed/concepts/multi-region/dual-region.md (hunk 5 above)
Comment (Langleu) [reply docs/self-managed/concepts/multi-region/dual-region.md]: I get your point about the arbitrary number, but we had before 25-30 minutes.
Just saying it takes whatever time for Backup and whatever time for Install sounds very fuzzy and questionable on why it can't be pinpoint.
I mean Elastic + WebApps have health checks in place, that's why there is a guaranteed lower bound anyway.
Sure there are more factors that can play in but I feel like giving 0 guidance is even less useful.

How about we rephase it to:

> - **failback** has an **RTO** of **5 + X** minutes to restore a functional state. Where X is the time it takes to back up and restore Elasticsearch, which is highly dependent on the setup and chosen [Elasticsearch backup type](https://www.elastic.co/guide/en/elasticsearch/reference/current/snapshots-register-repository.html#ess-repo-types).
During our automated tests, the reinstallation and reconfiguration of Camunda 8 takes 5 minutes. This can serve as a general guideline for the time required, though your experience may vary depending on your available resources and familiarity with the operational procedure.

Comment (deepthidevaki) [reply docs/self-managed/concepts/multi-region/dual-region.md]: The rephrase sounds good to me. :+1:

File: versioned_docs/version-8.5/self-managed/setup/deploy/amazon/amazon-eks/dual-region.md [hunk 11]
```diff
@@ -226,10 +226,10 @@ kubectl --context $CLUSTER_0 apply -f https://raw.githubusercontent.com/camunda/
 kubectl --context $CLUSTER_1 apply -f https://raw.githubusercontent.com/camunda/c8-multi-region/main/aws/dual-region/kubernetes/internal-dns-lb.yml
 ```
 
-2. Execute the script [generate_core_dns_entry.sh](https://github.com/camunda/c8-multi-region/blob/main/aws/dual-region/scripts/generate_core_dns_entry.sh) in the folder `aws/dual-region/scripts/` of the repository to help you generate the CoreDNS config. Make sure that you have previously exported the [environment prerequisites](#environment-prerequisites) since the script builds on top of it.
+2. Execute the script [generate_core_dns_entry.sh](https://github.com/camunda/c8-multi-region/blob/main/aws/dual-region/scripts/generate_core_dns_entry.sh) with the parameter `legacy` in the folder `aws/dual-region/scripts/` of the repository to help you generate the CoreDNS config. Make sure that you have previously exported the [environment prerequisites](#environment-prerequisites) since the script builds on top of it.
 
 ```shell
-./generate_core_dns_entry.sh
+./generate_core_dns_entry.sh legacy
```
[the remark this answers is not in the document]
Comment (Langleu) [reply versioned_docs/version-8.5/self-managed/setup/deploy/amazon/amazon-eks/dual-region.md:232]: I'll be able to remove it in the versioning rework again but atm for completeness would add it.

File: docs/self-managed/concepts/multi-region/dual-region.md [hunk 12]
```diff
@@ -54,8 +54,6 @@ The currently supported Camunda 8 Self-Managed components are:
```
[the remark this answers is not in the document]
Comment (Langleu) [reply docs/self-managed/concepts/multi-region/dual-region.md:54]: okay, now I understand what you mean about having suggestions on readability.
I'd suggest doing this in a separate PR decoupled from this topic, so DevEx and others can properly review your suggestions.

File: versioned_docs/version-8.5/self-managed/operational-guides/multi-region/dual-region-ops.md
[the remark this answers is not in the document]
Comment (Langleu) [reply versioned_docs/version-8.5/self-managed/operational-guides/multi-region/dual-region-ops.md:1]: Same as above, would suggest doing that in a separate PR decoupled from this topic.

**Comment by Langleu** [comment]
@conceptualshark could you do a last approval and feel free to merge afterwards!

I had to rebase the PR for the tests to pass.
Only changed since your approval:
- https://github.com/camunda/camunda-docs/pull/4205/commits/ad394f4e6135476b38b612c4190442b9478e45d8
- https://github.com/camunda/camunda-docs/pull/4205/commits/1ca09762fa978dd883f644ce820f89c8dc4fe559
